pandas/core/arrays/categorical.py

```diff
@@ -1436,62 +1426,6 @@ def check_for_ordered(self, op):
                 "Categorical to an ordered one\n"
             )
 
-    def _values_for_argsort(self):
-        return self._codes
-
-    def argsort(self, ascending=True, kind="quicksort", **kwargs):
```

Comment:
where did this doc-string go?

Reply:
got rid of it, can just use parent class docstring

Reply:
I think for this one we actually want to keep it

Reply:
just pushed restoring it

Reply:
restored+green